packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client/graphql/active-app-release.ts

```diff
@@ -5,6 +5,24 @@ export const ActiveAppReleaseQuery = gql`
   query activeAppRelease($appId: ID!) {
     app(id: $appId) {
       id
+      title
```

Comment:
These changes don't really make sense. The new API won't have any of these fields.